gapis/api/vulkan/api/types.api

```diff
@@ -73,3 +73,4 @@ type u32 VkSampleMask
 // Vulkan 1.1 core
 @replay_remap @nonDispatchHandle type u64 VkDescriptorUpdateTemplate
 @replay_remap @nonDispatchHandle type u64 VkSamplerYcbcrConversion
+@replay_custom_value type const VkAllocationCallbacks* AllocationCallbacks
```

Comment:
Looks like you've put this under Vulkan 1.1, when this is a Vulkan 1.0 thing?

Reply:
Done